packages/blade/src/components/Input/BaseInput/baseInputStyles.ts

```diff
@@ -46,60 +54,75 @@ export const getInputBackgroundAndBorderStyles = ({
   | 'isDropdownTrigger'
 >): CSSObject => {
   // normal state
-  let backgroundColor = theme.colors.interactive.background.gray.default;
-  let borderBottomColor = theme.colors.interactive.background.gray.highlighted;
+  let backgroundColor = getIn(theme.colors, baseInputBackgroundColor.default);
+  let borderColor = getIn(theme.colors, baseInputBorderColor.default);
+  let borderWidth = getIn(theme.border.width, baseInputBorderWidth.default);
 
-  // hoverState
-  if (isHovered) {
-    backgroundColor = theme.colors.interactive.background.gray.highlighted;
-  }
+  const baseInputState = isHovered
+    ? 'hovered'
+    : isFocused
+    ? 'focused'
+    : isDisabled
+    ? 'disabled'
+    : 'default';
 
-  // focused state
-  if (isFocused) {
-    backgroundColor = theme.colors.interactive.background.primary.faded;
-  }
+  backgroundColor = getIn(theme.colors, baseInputBackgroundColor[baseInputState]);
+  borderColor = getIn(theme.colors, baseInputBorderColor[baseInputState]);
+  borderWidth = getIn(theme.border.width, baseInputBorderWidth[baseInputState]);
 
-  // disabled state
-  if (isDisabled) {
-    backgroundColor = theme.colors.interactive.background.gray.disabled;
-    borderBottomColor = 'transparent';
-  }
-
-  // validation state
   if (validationState === 'error') {
-    backgroundColor = theme.colors.interactive.background.negative.faded;
-    borderBottomColor = theme.colors.interactive.background.negative.default;
+    borderColor = getIn(theme.colors, baseInputBorderColor.error);
   } else if (validationState === 'success') {
-    backgroundColor = theme.colors.interactive.background.positive.faded;
-    borderBottomColor = theme.colors.interactive.background.positive.default;
+    borderColor = getIn(theme.colors, baseInputBorderColor.success);
+    borderWidth = getIn(theme.border.width, baseInputBorderWidth.success);
   }
 
   return {
     backgroundColor,
-    borderBottomColor,
-    borderTopLeftRadius: makeBorderSize(theme.border.radius.small),
-    borderTopRightRadius: makeBorderSize(theme.border.radius.small),
-    borderBottomWidth: makeBorderSize(theme.border.width.thin),
-    borderBottomStyle: 'solid',
+    borderRadius: makeBorderSize(theme.border.radius.medium),
+    borderStyle: 'solid',
     display: 'flex',
     flexDirection: 'row',
     width: '100%',
     alignItems: isTextArea ? 'flex-start' : undefined,
     position: 'relative',
     height: isDropdownTrigger ? 'auto' : undefined,
+    border: 'none',
+    ...getBaseInputBorderStyles({ theme, borderColor, borderWidth, isFocused }),
   };
 };
 
+const inputPadding = {
```

Comment:
```suggestion
const inputPaddingTokens = {
```

just to convey that these are tokens? I thought they were pixel values in initial look

Reply:
Done